spec/takeaway_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "takeaway"
+
+describe Takeaway do
+
+  TEST_MENU = { "Won Ton" => 3.70,
+                "Chicken and Noodle Soup" => 1.90,
+                "Pork Chow Mein" => 4.60,
+              }
+
+  subject(:takeaway){described_class.new(menu: TEST_MENU, notification_class: notification_class)}
```

Comment:
Metrics/LineLength: Line is too long. [98/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)